pkg/virt-controller/services/template.go

```diff
@@ -96,6 +96,7 @@ type templateService struct {
 	virtLibDir                 string
 	ephemeralDiskDir           string
 	containerDiskDir           string
+	hotplugDiskDir             string
```

Comment:
Should there be a virt-launcher part to this as well? These other disk dirs are passed as cli args to virt-launcher.

Reply:
Yes, I totally missed that part, the latest push should incorporate it.